src/main/resources/view/MainWindow.fxml

```diff
@@ -35,47 +35,39 @@
           </Menu>
         </MenuBar>
 
-        <HBox prefWidth="400" style="-fx-background-color: #edede9">
-          <VBox styleClass="stack-pane">
-            <AnchorPane styleClass="split-pane">
-              <ImageView fx:id="logoImage" fitWidth="100.0" pickOnBounds="true" preserveRatio="true" AnchorPane.bottomAnchor="10.0" AnchorPane.leftAnchor="50.0" AnchorPane.rightAnchor="50.0" AnchorPane.topAnchor="10.0" />
-            </AnchorPane>
-            <StackPane fx:id="piechartPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
-            <StackPane fx:id="statisticsPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
+        <HBox HBox.hgrow="ALWAYS" prefWidth="400"  style="-fx-background-color: #f1e5e5">
+          <VBox styleClass="stack-pane" HBox.hgrow="ALWAYS">
+            <StackPane styleClass="stack-pane">
+              <padding>
+                <Insets top="10" />
+              </padding>
+              <ImageView fx:id="logoImage" fitWidth="200.0" pickOnBounds="true" preserveRatio="true"/>
+            </StackPane>
+            <StackPane fx:id="statisticsPlaceholder" prefWidth="200.0" maxHeight="100.0" styleClass="card"/>
+            <StackPane fx:id="rsvpPlaceholder" prefHeight="100.0" prefWidth="200.0" maxHeight="100.0" styleClass="card" />
+            <StackPane fx:id="dietaryPlaceholder" prefHeight="200.0" prefWidth="200.0" styleClass="card" />
           </VBox>
-          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS">
-            <padding>
-              <Insets bottom="10" left="10" right="10" top="10" />
-            </padding>
+          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS" HBox.hgrow="ALWAYS">
             <StackPane fx:id="guestListPanelPlaceholder" VBox.vgrow="ALWAYS" />
```

Comment:
The spacing on top of guestlist / vendorlist is inconsistent with the spacing between elements used throughout the rest of the app. It is narrower and thus seems slightly cramped. Would it be possible to make the spacing the same as every other place (e.g. the spacing between total count and rsvp status panel, and rsvp status and dietary panel)